ros_bt_py/ros_bt_py/tree_manager.py

```diff
@@ -415,10 +413,19 @@ def __init__(
                 1.0 / diagnostics_frequency, self.diagnostic_callback
             )
 
-    def get_state(self):
+    @typechecked
+    def get_state(self) -> str:
         with self._state_lock:
             return self.tree_msg.state
 
+    @typechecked
+    def set_state(self, new_state: str) -> None:
+        with self._state_lock:
+            self.tree_msg.state = new_state
+            self.ros_node.get_logger().fatal(
+                f"Updating tree state to {self.tree_msg.state}"
+            )
+
```

Comment:
I like this fix, but I still found some lines with the old structure in the file. Is that on purpose?

Reply:
No I changed these occurences.